dropwizard-testing/src/main/java/io/dropwizard/testing/junit/ResourceTestJerseyConfiguration.java

```diff
@@ -0,0 +1,39 @@
+package io.dropwizard.testing.junit;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.glassfish.jersey.test.spi.TestContainerFactory;
+
+import javax.validation.Validator;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * A configuration of a Jersey testing environment.
+ * It's used for configuring {@link DropwizardTestResourceConfig} from {@link ResourceTestRule}
```

Comment:
I think this statement could be made a bit more clear for those who aren't familiar with the code it refers to. How about "Encapsulates data required to configure a {@link ResourceTestRule}. Primarily accessed via {@link DropwizardTestResourceConfig}."